.pylintrc

```diff
@@ -4,4 +4,7 @@ disable=
     missing-class-docstring,
     missing-function-docstring,
     bad-continuation, # Doesn't like black formatter
-    unsubscriptable-object # Gives false positives for typing
\ No newline at end of file
+    unsubscriptable-object, # Gives false positives for typing
+    too-many-instance-attributes,
+    consider-using-in,
+    duplicate-code
```

Comment:
When you globally disable certain lint errors, you should specify the reason in a comment.